### ISSUE TITLE
False positive for 303 when using command module for setting system runlevel

### ISSUE THREAD

**Issue by badgateway666** [ansible-community/ansible-lint#682]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version: ansible 2.9.4
ansible-lint --version: ansible-lint 4.1.0
```

- ansible installation method: OS package
- ansible-lint installation method: pip

# Desired Behaviour

ansible-lint should recognize that systemctl is not used for enabling/disabling services,
but for setting the system runlevel:
```
- name: Set runlevel
  command: systemctl set-default multi-user.target
```

Ideally, there would also be a module for managing runlevels with systemd

# Actual Behaviour (Bug report only)

`[303] systemctl used in place of systemd module
/home/user/Projects/ansible/roles/common/tasks/main.yml:9
Task/Handler: Set runlevel
